### PR TITLE
Mention llvm fork in libraries forked from llvm-project.

### PR DESCRIPTION
**Pull request by sbc100**
I created a fork of llvm-project to hold our fork and make
merging new changes easier.  Use that to document the changes
rather than trying to keep these readme files up-to-date.

### CODE REVIEW

File: system/lib/compiler-rt/readme.txt [hunk 1]
```diff
@@ -14,4 +14,8 @@ Run `system/lib/update_compiler_rt.py path/to/llvm-root`
 Local Change
 ------------
 
-lib/builtins/powitf2.c: enable for wasm as well as PPC
+The differences from upstream compiler-rt are stored in our fork of llvm.  The
+changes from upstream can seen by comparing the `llvmorg-10.0.0` and
+emscripten-libs-10.0.0` branches:
```
Comment (kripken) [inline system/lib/compiler-rt/readme.txt:19]: The actual location of the fork may be good to mention? I had to look for it for a while until I saw it was embedded in the comparison URL below.
```suggestion
The differences from upstream compiler-rt are stored in our fork of llvm (which
is at `emscripten-core/llvm-project`).

Changes from upstream can seen by comparing the `llvmorg-10.0.0` and
emscripten-libs-10.0.0` branches:
```

**Review by kripken — APPROVED** [review]

File: system/lib/compiler-rt/readme.txt (hunk 1 above)
Comment (sbc100) [reply system/lib/compiler-rt/readme.txt:19]: OK, I actually ended up re-worded. WDYT.

File: system/lib/compiler-rt/readme.txt [hunk 2]
```diff
@@ -1,4 +1,14 @@
-These files are from llvm-project/compiler-rt, release 10.0.0.
+llvm's compielr-rt
+------------------
+
+These files are from the llvm-project based on release 10.0.0.
+
+We maintain a local fork of llvm-project that contains any emscripten
+specific patches:
+
+  https://github.com/emscripten-llvm-project
```
Comment (kripken) [inline system/lib/compiler-rt/readme.txt:9]:
```suggestion
  https://github.com/emscripten-core/llvm-project
```

Comment (sbc100) [reply system/lib/compiler-rt/readme.txt:9]: Done

File: system/lib/compiler-rt/readme.txt [hunk 3]
```diff
@@ -1,4 +1,14 @@
-These files are from llvm-project/compiler-rt, release 10.0.0.
+llvm's compielr-rt
```
Comment (kripken) [inline system/lib/compiler-rt/readme.txt:1]:
```suggestion
llvm's compiler-rt
```

**Review by kripken — APPROVED** [review]

**Pull request merged**